src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -5,10 +5,20 @@
 import core.basesyntax.model.User;
 
 public class RegistrationServiceImpl implements RegistrationService {
+    private static final int DEFAULT_MIN_CHARACTERS = 6;
     private final StorageDao storageDao = new StorageDaoImpl();
 
     @Override
     public User register(User user) {
-        return null;
+        if (user.getLogin().length() < DEFAULT_MIN_CHARACTERS
+                || user.getPassword().length() < DEFAULT_MIN_CHARACTERS
+                || user.getAge() < 18) {
```

Comment:
What if user.getAge() or user.getPassword() or user.getLogin == null?

And maybe it will be better if you separate them, then you will know what exact part of user is lost
"Don’t create one big if with all conditions. Better to split one big if into several small ones"